tools/lint/tests/test_file_lints.py

```diff
@@ -633,8 +633,9 @@ def test_late_timeout():
             ]
 
 
```

Comment:
Can you leave a short comment to avoid future-us being silly again? Something like.

`# The print function - print() - is checked in test_print_function below.`